app/js/controllers/dev.js

```diff
@@ -0,0 +1,21 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import CameraLatestImageView from 'js/views/dev/camera-latest-image';
+
+export default class DevController extends Controller {
```

Comment:
> I understand the point, and basically as a first hacky approach for having a demo for the next 24 hours could be valid. Period.
TLDR: We all agree, this is a temporary solution to show case what we can do with the app. We need more thinking.

True, I'm just trying to understand why do we want to mimic kind of final and generic approach with "ServiceGetterController or GetterController" and not to go with temporal anything temporal that I have now